node/tests/integration_tests.rs

```diff
@@ -23,7 +23,7 @@ async fn setup_peer(path: &str, peers: &[Peer]) {
     let mut file = tokio::fs::File::create(format!("{path}/peers.json"))
         .await
         .unwrap();
-    file.write_all(&serde_spb::to_vec(&peers).unwrap())
+    file.write_all(serde_spb::to_string(&peers).unwrap().as_bytes())
```

Comment:
No need to fix other parts of the code where it uses `serde_spb::to_vec`?

Reply:
nope. this is the only part where the code assumes that `to_vec()` and `to_string()` use the same foramt. (which is not from now)